doc/integrator/database.rst

```diff
@@ -39,12 +50,12 @@ Starting point is that the current version is the same on integration and produc
 Prepare the project
 ~~~~~~~~~~~~~~~~~~~
 
-To be able to proceed like this, the variables ``PGSCHEMA`` and the ``DOCKER_PGSCHEMA_STATIC``
-should be managed in your makefiles:
+To be able to proceed like this, the variables ``PGSCHEMA`` and the ``PGSCHEMA_STATIC``
+should be managed in your make files:
```

Comment:
Are we still using makefiles, I thought we now use env files for this?

Reply:
Oups, ..., thanks :-)